css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        flex-basis: calc(100% / (1 / 6));
```

Comment:
`flex-basis` is a child property which means it needs to get applied to children of flex parents

Reply:
pushed the new file and here's the new screenshot:
![Screenshot 2024-12-19 at 6 56 36 PM](https://github.com/user-attachments/assets/ab3719b8-2ba9-40be-a8fd-78dc6e893fa9)